static/src/js/util/request/exportCollectionSearchRequest.js

```diff
@@ -0,0 +1,9 @@
+import ExportSearchRequest from './exportSearchRequest'
+
+export default class ExportCollectionSearchRequest extends ExportSearchRequest {
+  constructor(authToken, earthdataEnvironment) {
+    super(authToken, earthdataEnvironment)
+
+    this.searchPath = 'collections/export'
```

Comment:
Why does granule search have different paths for authenticated requests but collections don't?

Reply:
We chatted about this in the review